src/patterns/PetCard/PetCard.html

```diff
@@ -12,14 +12,20 @@
     </span>
     <div v-if="pet.status == 'adopted'"
       class="border-bottom blue">Adopted</div>
-    <div v-else-if="pet.specialNeeds || pet.bondedTo || pet.actQuickly"
+    <div v-else-if="pet.actQuickly"
       class="border-bottom orange">Act Quickly</div>
   </a>
   <div class="content">
     <Heading level="h4"
       font-weight="bold">{{ pet.name }}</Heading>
     <Paragraph font-size="s">
-      {{ petFormatted.sex }}, {{ petFormatted.age }}<span v-if="pet.specialNeeds">, <span class="orange">Special Needs</span></span><span v-if="pet.specialNeeds">, <span class="orange">Bonded Pair</span></span>
+      {{ petFormatted.sex }}, {{ petFormatted.age }}
```

Comment:
I think this reformatting of the code actually added extra space to the text in the pet card.  Now there is a space before and after the comma.   I added an image in the ticket.  It is also visible in periodic